opentelemetry-exporter-google-cloud/src/opentelemetry/exporter/cloud_monitoring/__init__.py

```diff
@@ -186,17 +189,23 @@ def _get_metric_descriptor(
             descriptor["metric_kind"] = MetricDescriptor.MetricKind.CUMULATIVE
         elif isinstance(record.aggregator, ValueObserverAggregator):
             descriptor["metric_kind"] = MetricDescriptor.MetricKind.GAUGE
+        elif isinstance(record.aggregator, HistogramAggregator):
+            descriptor["metric_kind"] = MetricDescriptor.MetricKind.CUMULATIVE
```

Comment:
histogram is a monotonic counter for every bucket - so the metric kind is the same as counter.

Reply:
That makes sense. Is non-stateful Histogram supported in OT python and would that need to be Gauge?

Reply:
no, just like Counter, a non-stateful histogram is a cumulative but the start time is moved to the previous export time instead of always being at the start time of the program